src/lib/concurrency/transaction_context.cpp

```diff
@@ -10,11 +10,13 @@
 
 namespace opossum {
 
-TransactionContext::TransactionContext(const TransactionID transaction_id, const CommitID snapshot_commit_id)
+TransactionContext::TransactionContext(const TransactionID transaction_id, const CommitID snapshot_commit_id,
```

Comment:
(Random line)
Have you tried adding a couple of SQL strings with begin/commit/rollback to sqlite_testrunner_queries.sql? While no replacement for unit tests, that should make it easy to further improve the test coverage.

Reply:
I added some transaction statements to the sqlite test runner now and the tests pass. Let us know if you want us to add some more queries.

Reply:
Yes, we have now added some SQL queries containing transaction statements to the sqlite test runner and the tests pass.